filter/data.go

```diff
@@ -41,9 +41,10 @@ type DimensionOptions struct {
 
 // CreateBlueprint represents the fields required to create a filter blueprint
 type CreateBlueprint struct {
-	Dataset    Dataset          `json:"dataset"`
-	Dimensions []ModelDimension `json:"dimensions"`
-	FilterID   string           `json:"filter_id"`
+	Dataset        Dataset          `json:"dataset"`
+	Dimensions     []ModelDimension `json:"dimensions"`
+	FilterID       string           `json:"filter_id"`
+	PopulationType string           `json:"population_type"`
```

Comment:
Should `PopulationType` has omitempty? This method is used my CMD
```suggestion
	PopulationType string          `json:"population_type,omitempty"`
```

Reply:
Indeed it should, I've now added it.